src/antares/service/api_services/link_api.py

```diff
@@ -160,20 +163,57 @@ def update_link_ui(self, link: Link, ui: LinkUi) -> LinkUi:
             link_ui = LinkUi.model_validate(json_response)
 
         except APIError as e:
-            raise LinkUiUpdateError(link.name, e.message) from e
+            raise LinkUiUpdateError(link.id, e.message) from e
 
         return link_ui
 
-    def read_links(self) -> list[Link]:
-        raise NotImplementedError
+    def get_parameters(self, area_from_id: str, area_to_id: str) -> pd.DataFrame:
```

Comment:
We should choose between area_from_id and area_from between the 2 methods. I think we should type area_from_id as it's clearer